vignettes/userguide.Rmd

```diff
@@ -189,6 +189,42 @@ options(
 
 We recommend that you [set up SSH keys](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server) for password-less login.
 
+### Options
+
+The various configurable options are mentioned throughout the documentation where applicable, however, we list all of the options here for reference.
+
+Options can be set by including a call to `options()` in your `.Rprofile`, or by calling `options()` in a script or interactively during a session.
+
+- `clustermq.scheduler` 
+  + One of the supported `clustermq` schedulers; options are `"LOCAL"`, `"multiprocess"`, `"multicore"`, `"lsf"`, `"sge"`, `"slurm"`, `"pbs"`, `"Torque"`, or `"ssh"` (default is `qsys_default`, which is usually `"LOCAL"`)
+
+- `clustermq.host` 
+  + The name of the node or device for constructing the `ZeroMQ` host address (default is `Sys.info()["nodename"]`)
+
+- `clustermq.ssh.host`
+  + The user name and host for connecting to the HPC via SSH (e.g. `user@host`); we recommend setting up SSH keys for password-less login.
+
+- `clustermq.ssh.log`
+  + Path for a file that will be created and populated with logging information regarding the SSH connection (e.g. `"~/cmq_ssh.log"`); helpful for debugging purposes
+
+- `clustermq.ssh.timeout` 
+  + The amount of time to wait (in seconds) for a SSH startup connection before timing out (default is 5 seconds)
+
+- `clustermq.worker.timeout` 
+  + The amount of time to wait (in seconds) for master-worker communication before timing out (default is 600 seconds)
+
+- `clustermq.error.timeout` 
+  + The amount of time to wait (in seconds) after an error occurs before timing out (default is `min(timeout, 30)` seconds)
+
+- `clustermq.template` 
+  + Path to a template file for submitting HPC jobs; only necessary if using your own template, otherwise the default template will be used (default depends on `clustermq.scheduler`)
+
+- `clustermq.data.warning` 
+  + The threshold for the size of the common data (in Mb) before `clustermq` throws a warning (default is 1000)
+
+- `clustermq.defaults` 
+  + A list object containing any number of `clustermq` options described above (default is an empty list (i.e. `list()`))
```

Comment:
A named list of default template values (not for the options listed here). This takes precedence of defaults specified in the template file